src/jsdoc_transformer.ts

```diff
@@ -173,22 +174,22 @@ export function maybeAddHeritageClauses(
           return null;
         }
       }
-    } else if (sym.flags & ts.SymbolFlags.Value) {
+    } else if (trueSym.flags & ts.SymbolFlags.Value) {
       // If it's something other than a class in the value namespace, then it will
       // not be a type in the Closure output (because Closure collapses
       // the type and value namespaces).
       mtt.debugWarn(
           decl, `omitting heritage reference to a type/value conflict: ${expr.getText()}`);
       return null;
-    } else if (sym.flags & ts.SymbolFlags.TypeLiteral) {
+    } else if (trueSym.flags & ts.SymbolFlags.TypeLiteral) {
       // A type literal is a type like `{foo: string}`.
       // These can come up as the output of a mapped type.
       mtt.debugWarn(decl, `omitting heritage reference to a type literal: ${expr.getText()}`);
       return null;
     }
 
     // typeToClosure includes nullability modifiers, so call symbolToString directly here.
-    const parentName = typeTranslator.symbolToString(sym);
+    const parentName = typeTranslator.symbolToString(localSym);
```

Comment:
I'm still wondering about the pattern though where when we `forwardDeclare` (or soon: `requireType`) a module, we're building a mapping from the symbols as declared by the module to unique names. With this here, we're looking up by local symbol to find the name alias. Given tests look green, maybe I'm misunderstanding what symbol the import is pointing to, and all is fine? This might need attaching a debugger and checking the object identity of the symbols involved...